newIDE/app/src/UI/Dialog/index.js

```diff
@@ -39,7 +39,7 @@ type Props = {|
   secondaryActions?: React.Node,
   onRequestClose?: () => void,
 
-  modal?: boolean, // Force the user to use one of the actions in the Dialog. Clicking outside the Dialog will not trigger the onRequestClose.
+  modal?: boolean, //Force the user to use one of the actions in the Dialog. Clicking outside the Dialog will not trigger the onRequestClose. Hence, Dialog will not close if clicked out of its scope
```

Comment:
`// If true, the dialog can't be closed by clicking outside or pressing Escape.`

Reply:
modified